crates/shared/src/price_estimation.rs

```diff
@@ -138,8 +138,8 @@ impl FromStr for PriceEstimator {
     }
 }
 
-#[derive(Clone, Debug)]
-pub struct NativePriceEstimators(Vec<NativePriceEstimator>);
+#[derive(Clone, Debug, Display)]
```

Comment:
You can derive `Display`? Woah! What does it look like?

Reply:
**Edit**: I think this may be a typo - there is a manual `Display` implementation 👇.